deploy/eks/action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'eks-deploy'
+name: "eks-deploy"
```

Comment:
nit: Do we gain anything by converting these to double quotes or is it just personal preference?
I was wondering when to use `""` vs `''` vs none in CI yml files in general
(see description: fields below for an example that doesn't have any quotes)

No need to action on this, it is more for my education.

Reply:
This is just Prettier going crazy on the file. I think there should be no quotes given the convoluted YAML string interpretations. I'll touch this up.

Reply:
I noticed GitHub uses single quotes for titles/descriptions, double quotes for `using` etc... I'm thinking if it works, and it's not a security risk, there are higher value things to focus our attention on and I can just leave this a bit butchered until someone's better linter fixes it later 😂 

You good with that? I'll fix if you think it's worthwhile.

Reply:
Nope definitely good to leave it as is!